packages/create/templates/hello-v2/src/operations.ts

```diff
@@ -0,0 +1,64 @@
+// Welcome to DBOS!
+
+// This is a sample "Hello" app built with DBOS.
+// It greets visitors and keeps track of how many times each visitor has been greeted.
+
+// First, let's import DBOS
+import { DBOS } from "@dbos-inc/dbos-sdk";
+
+// Then, let's declare a type representing the "dbos_hello" database table
+export interface dbos_hello {
+  name: string;
+  greet_count: number;
+}
+
+// Now let's define a class with some static functions.
+// DBOS uses TypeScript decorators to automatically make your functions reliable, so they need to be static.
+export class Hello {
+  // Serve this function from HTTP GET requests at the /greeting endpoint with 'user' as a path parameter
+  @DBOS.getApi("/greeting/:user")
```

Comment:
If we want to eventually get rid of `getApi` and other HTTP verbs, we should use Koa directly here.

Reply:
Disagree strongly.

Reply:
We can discuss this some other time -- maintaining a thin wrapper around Koa is pretty annoying. Remember the times we wanted to support CORS and preflight requests :/

Reply:
Agree with one part of the Koa point (features have to be added in the framework), strong disagree on the CORS part, baking that into your code is bad and should be managed by the deployment, which we can do better if we help you in a "serverless" way.